source/security/minio-authentication-authorization.rst

```diff
@@ -560,8 +573,8 @@ The following table lists the MinIO-supported policy action keys.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 MinIO supports the following actions for use with defining policies
-for :mc-cmd:`mc admin` operations. These actions are *only* valid for
-MinIO deployments and are *not* intended for use with other S3-compatible
+for :mc-cmd:`mc admin` operations. These actions are **only** valid for
```

Comment:
What is mc-cmd here?  Is this referencing an alias to mc or is it a formatting element?

Reply:
Yep - in Sphinx, `:<string>:` is a directive, which is equivalent to a function. In this case, the `:mc-cmd:` directive searches through an index of `mc-cmd` objects for one that matches the string `mc admin` and links to the location of that object.

TL;DR if you click the `mc admin` text, it links to the page for `mc admin`. No hardlinking required.